app/lib/features/chat/widgets/messages/redacted_message.dart

```diff
@@ -6,11 +8,28 @@ class RedactedMessageWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
     return Container(
-      padding: const EdgeInsets.all(18),
-      child: Text(
-        L10n.of(context).chatMessageDeleted,
-        style: Theme.of(context).textTheme.bodySmall,
+      padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 8),
+      child: Text.rich(
```

Comment:
that gave me pause and I checked again, but this is just a copy from the other widget, isn't it? We can just use a `Row([Icon(..), Text(..)]` here, can't we? If so, I think we should.